include/deal.II/fe/fe_values_extractors.h

```diff
@@ -301,6 +327,38 @@ namespace FEValuesExtractors
   inline Tensor<rank>::Tensor(const unsigned int first_tensor_component)
     : first_tensor_component(first_tensor_component)
   {}
+
+
+  inline std::string
+  Scalar::get_name() const
```

Comment:
Not wrong, but could probably also be in the `.cc` file to make compiling everything else faster. I suspect that these functions aren't performance critical.